modtools/create-unit.lua

```diff
@@ -25,7 +25,6 @@ local usage = [====[
 modtools/create-unit
 ====================
 Creates a unit.  Usage::
-
```

Comment:
Oh, it might complain about lines that are just whitespace (e.g. a bunch of spaces on their own line). Anyway, I can take care of whatever Travis is complaining about, so don't feel like you have to make a bunch of commits to fix that.